package.json

```diff
@@ -84,7 +86,9 @@
         "sinon-chai": "^3.4.0",
         "tslint": "^6.0.0",
         "typescript": "^3.7.5",
-        "vscode-test": "^1.3.0"
+        "vscode-test": "^1.3.0",
+        "vscode-extension-tester": "^2.4.0",
```

Comment:
it is better to try to keep the list of dependencies in alphabetical order. it is easier to find one when reading